python/tvm/relax/vm_build.py

```diff
@@ -249,6 +248,7 @@ def build(
     mod: tvm.IRModule,
     target: Union[str, tvm.target.Target],
     params: Optional[Dict[str, list]] = None,
+    pipeline: str = "default",
```

Comment:
I appreciate this PR's work, but I am still concerned about setting the "default" pipeline to the `build` API. 
As we said we'd like to keep it a "minimal" build. It would be great to set the default value to None, while give more time for the community to discuss about the default pipeline in `build` API